examples/animation/resources/scene.gltf

```diff
@@ -0,0 +1,3151 @@
+{
```

Comment:
You should add

```
*.gltf -diff
```
to the `.gitattributes` file (or make it if it doesn't exist) so these are treated as binary.

Reply:
I didn't see one, so I just created it